VideoApp/VideoApp/NewCode/Screens/MediaSetup/MediaSetupViewModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright (C) 2022 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Combine
+
+class MediaSetupViewModel: ObservableObject {
+    @Published var participant = ParticipantViewModel()
```

Comment:
Kind of an awkward naming situation because I have 2 different types related to local participant. But this seemed like the least bad option. At least it is nice for the UI.